test/services/PayeAtsServiceSpec.scala

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services
+
+import connectors.MiddleConnector
+import models.PayeAtsData
+import org.mockito.Matchers.{any, eq => eqTo}
+import org.mockito.Mockito.when
+import org.scalatest.concurrent.{IntegrationPatience, ScalaFutures}
+import org.scalatest.mockito.MockitoSugar
+import org.scalatestplus.play.guice.GuiceOneAppPerTest
+import play.api.libs.json.{JsResultException, JsValue, Json}
+import uk.gov.hmrc.http.{BadRequestException, HeaderCarrier, HttpResponse, NotFoundException}
+import uk.gov.hmrc.play.test.UnitSpec
+import utils.JsonUtil
+import utils.TestConstants.testNino
+
+import scala.concurrent.Future
+import scala.io.Source
+
+class PayeAtsServiceSpec extends UnitSpec with MockitoSugar with JsonUtil with GuiceOneAppPerTest with ScalaFutures with IntegrationPatience {
+
+  implicit val hc = HeaderCarrier()
+  val expectedResponse: JsValue = readJson("/paye_ats.json")
+  private val currentYear = 2018
+
+  private def readJson(path: String) = {
```

Comment:
Im a little confused why we have both this function and the JsonUtil trait inherited. Is this function just for the purpose of loading it as a Stream? If so, can we remove the JsonUtil. If we dont care about it being a stream, is there a reason why we arent using the functions in JsonUtil?